frontend/src/utils/account-send.ts

```diff
@@ -0,0 +1,144 @@
+import { isAddress, keccak256, toUtf8Bytes, BigNumber, hexZeroPad } from 'src/utils/ethers';
+import localforage from 'localforage';
+import { toAddress, lookupAddress } from 'src/utils/address';
+import { LOCALFORAGE_ACOUNT_SEND_KEY } from 'src/utils/constants';
+import { Web3Provider } from 'src/utils/ethers';
+
+type AccountData = {
+  address: string;
+  advancedMode: boolean;
+  checkbox: boolean;
+};
+
+type KeyData = {
+  count: number;
+  viewingKey: string;
+};
+
+type EncryptedData = {
+  encryptedAddress: string;
+};
+
+type AccountSendData = {
+  amount: string;
+  tokenAddress: string;
+  dateSent: Date;
+  hash: string;
+  userAddress: string;
+  recipientAddress: string;
+  checkbox?: boolean;
```

Comment:
What does `checkbox` mean? And above, what is the `hash` a hash of? Let's add comments explaining the params that are not intuitive by name. `count` fits here too I'd say

(edit: oh ok, advanced mode bit is the generic setting, checkbox is the "use pubkey" checkbox. yea let's clarify a bit here)